appnav/src/main/kotlin/io/element/android/appnav/room/RoomFlowNode.kt

```diff
@@ -116,14 +142,27 @@ class RoomFlowNode @AssistedInject constructor(
 
     override fun resolve(navTarget: NavTarget, buildContext: BuildContext): Node {
         return when (navTarget) {
-            NavTarget.Loading -> loadingNode(buildContext)
-            NavTarget.JoinRoom -> {
-                val inputs = JoinRoomEntryPoint.Inputs(inputs.roomId, roomDescription = inputs.roomDescription)
+            is NavTarget.Loading -> loadingNode(buildContext)
+            is NavTarget.Resolving -> {
+                val callback = object : RoomAliasResolverNode.Callback {
+                    override fun onAliasResolved(roomId: RoomId) {
+                        backstack.newRoot(NavTarget.JoinRoom(roomId))
```

Comment:
you should call `subscribeToRoomInfoFlow(roomId: RoomId) `instead of pushing `JoinRoom`

Reply:
You're right, good catch! Done.